src/App.js

```diff
@@ -1,10 +1,26 @@
+import TodoBox from './components/TodoBox';
+import styled from 'styled-components';
+import GlobalStyle from './GlobalStyle';
 
-function App() {
+const TodoContainer = styled.div`
+  margin: 0 auto;
+  margin-top: 3rem;
+  background: rgba(189, 187, 187, 0.8);
+  width: 30rem;
+  height: 80vh;
+  border-radius: 30px;
+  box-shadow: 0 5px 18px -7px rgba(0, 0, 0, 1);
+  padding: 30px;
+`;
+
+const App = () => {
   return (
-    <div>
-      <h1>15기 화이팅~</h1>
-    </div>
+    <>
+      <GlobalStyle />
+      <TodoContainer>
+        <TodoBox></TodoBox>
```

Comment:
태그가 아무것도 감싸고 있지 않은 경우 이렇게 self-closing 형태로 사용해 볼 수 있습니다
```suggestion
        <TodoBox />
```

Reply:
처음에 App.js에 모든걸 끌어와서 구현하던 흔적이 남았었군요.. 수정하겠습니다!!